cfgov/ask_cfpb/models/pages.py

```diff
@@ -94,12 +101,44 @@ def get_ask_breadcrumbs(category=None):
     return breadcrumbs
 
 
+def get_journey_breadcrumbs(request, path):
+    """
+    If referrer is a BAH journey page, breadcrumbs should
+    reflect the BAH journey page hierarchy.
+    """
+    pages = path.strip('/').replace('process/', '').split('/')
+    if len(pages) == 1:
```

Comment:
if len(pages) == 1, there will be no pages[1]
do you want to add a value to pages with something like this?
```
if len(pages) == 1:
    pages.append('prepare') 
```

Reply:
@higs4281 Sorry this is cryptic! We wound up not creating a page at `/process/` in the Wagtail tree when we migrated the journey pages, but are still using URLs with that segment, which has necessitated a number of temporary workarounds. This strips `/process/` from the referring path, and then, if the user came from `/owning-a-home/process`, adds back the slug for the first journey page.

I'll add a comment that this bit of code also needs to be removed when we can switch to the new `process`-less URLs after the homebuying season campaign ends.

Reply:
thanks for the explanation, @virginiacc

Reply:
@higs4281 Thanks for catching the assignment issue -- should be fixed now!